index.js

```diff
@@ -132,14 +189,39 @@ function testGenOperation(swagger, path, operation, config) {
  * @returns {string|Array} set of all tests for a path
  */
 function testGenPath(swagger, path, config) {
-  var operations = swagger.paths[path];
+  var parameters = {
+    queryParameters: [],
+    bodyParameters: [],
+    headerParameters: [],
+    formParameters: [],
+    pathParameters: []
+  };
+  var childProperty = swagger.paths[path];
   var result = [];
-  var op;
+  var cp;
+  var p;
   var validOps = ['get', 'put', 'post', 'delete', 'options', 'head', 'patch'];
 
-  for (op in operations) {
-    if (operations.hasOwnProperty(op) && validOps.indexOf(op) >= 0) {
-      result.push(testGenOperation(swagger, path, op, config));
+  if (childProperty.hasOwnProperty('parameters')) {
+    // only adds body parameters to request, ignores query params
+    for (p in childProperty.parameters) {
```

Comment:
@mohsen1 @elsapeng 
Currently, we are enforcing variable declarations on the outside of for-in loops. If there is a preference for something else, we can open an issue and fix all of them